### ISSUE TITLE
get bug in testing sample_xwt.py

### ISSUE THREAD

**Issue by lovechang1986** [nabobalis/pycwt#6]
"C:\Anaconda\lib\site-packages\pycwt-0.5.dev0-py2.7.egg\pycwt\wavelet.py", line 101, in cwt
    freqs = 1 / (wavelet.flambda() * sj)
AttributeError: 'unicode' object has no attribute 'flambda'.

what's the mean of 'flambda'? 
i can't found the definition of 'flambda'.

**Comment by nabobalis** [comment]
Yeah, I reverted some code to make it work under Python 3 and did not test it make sure it did not break under python 2. 

Give me a bit to patch this issue, under python 2 it does not create the mother object and leaves it as a string.

**Comment by lovechang1986** [comment]
O..Thanks.
I will try python3.

**Comment by nabobalis** [comment]
Well it should work under both versions! I have no plans to drop Python 2 just yet.

**Comment by lovechang1986** [comment]
:+1:

**Comment by nabobalis** [comment]
I just added a new commit for now, I ran the sample scripts under py2 and py3 and they worked. So hopefully this is fixed.

**Comment by lovechang1986** [comment]
:+1: .
Another issue i found.
some scripts line using like :      W1, sj, freq, coi, _, _ = cwt(signal/std1, **cwt_kwargs).
But in wavelet.py the function 'cwt' just return four parameters?

**Comment by nabobalis** [comment]
Ah, interesting. I think i need to rego over this code someday. I think for now, it would be better to use the main copy from https://github.com/regeirk/pycwt.
